fees/move-dollar.ts

```diff
@@ -0,0 +1,40 @@
+import fetchURL from "../utils/fetchURL";
+import { SimpleAdapter } from "../adapters/types";
+import { CHAIN } from "../helpers/chains";
+
+const thalaDappURL = 'https://app.thala.fi/';
+const feesQueryURL = `${thalaDappURL}/api/protocol-fee-chart?timeframe=`;
+
+const feesEndpoint = (startTimestamp: number, timeframe: string) => 
+startTimestamp ? feesQueryURL + timeframe + `&startTimestamp=${startTimestamp}` : feesQueryURL + timeframe;
```

Comment:
is can query `start` and `end` time, if you set like this it can not backfill data

Reply:
have made modifications so backfill is now possible - endTimestamp is the date which is considered, and timeframe is how far behind that timestamp we set our range to be